zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/DecisionInstanceQueryControllerTest.java

```diff
@@ -380,6 +381,70 @@ void shouldSearchVariablesWithAdvancedFilter(
         .search(new DecisionInstanceQuery.Builder().filter(filter).build());
   }
 
+  @Test
+  void shouldThrowExceptionWithInvalidDateTime() {
+    // given
+    final var request = "{\"filter\": {\"evaluationDate\": \"invalid\"}}";
+
+    // when / then
+    final var expectedResponse =
+        """
+            {
+              "type": "about:blank",
+              "title": "Bad Request",
+              "status": 400,
+              "detail": "Failed to parse date-time: [invalid]",
+              "instance": "/v2/decision-instances/search"
+            }""";
+    webClient
+        .post()
+        .uri("/v2/decision-instances/search")
+        .accept(MediaType.APPLICATION_JSON)
+        .contentType(MediaType.APPLICATION_JSON)
+        .bodyValue(request)
+        .exchange()
+        .expectStatus()
+        .isBadRequest()
+        .expectHeader()
+        .contentType(MediaType.APPLICATION_PROBLEM_JSON)
+        .expectBody()
+        .json(expectedResponse);
+
+    verify(decisionInstanceServices, never()).search(any(DecisionInstanceQuery.class));
+  }
+
+  @Test
+  void shouldThrowExceptionWithWrongType() {
+    // given
+    final var request = "{\"filter\": {\"evaluationDate\": []}";
+
+    // when / then
+    final var expectedResponse =
+        """
+            {
+              "type": "about:blank",
+              "title": "Bad Request",
+              "status": 400,
+              "detail": "Failed to read request",
```

Comment:
❓ Is that the default message from Jackson/Spring? That is quite generic and won't help the user much, I assume 🙈 Can we do any better? 😄

Reply:
The location/field that cannot be read would help already, I assume.

Reply:
Yes, that is just the default. I'll look into that, we could probably add a Spring exception handler for it.

Reply:
Changed it to `Request property [%s] cannot be parsed`.